Configure

```diff
@@ -3249,6 +3257,30 @@ sub absolutedir {
     return realpath($dir);
 }
 
+# Check if all paths are one and the same, using stat.  They must both exist
+# We need this for the cases when File::Spec doesn't detect case insensitivity
+# (File::Spec::Unix assumes case sensitivity)
+sub samedir {
+    my @stat;
+
+    foreach (@_) {
+        die "$_ doesn't exist" unless -e $_;
+
+        my @stat2 = stat($_);
+        unless (@stat) {
+            @stat = @stat2;
+            next;
+        }
```

Comment:
This looping construct seems like a particularly opaque way of getting the stat of the two arguments. Surely just calling stat twice (once for each argument) is much simpler? I had to stare at this code for several minutes trying to figure out WTF was going on.

Reply:
Sure, I can do that...  that makes it less generic, but ok

Reply:
Done